http-streaming/README.md

```diff
@@ -0,0 +1,297 @@
+
+1. File- or object-based, e.g., S3 access
+2. Database access, e.g., direct Snowflake access
+3. Decoupled client-server APIs, e.g., REST APIs, gRPC, webhooks, and streaming APIs. In this context, "decoupled" means that the client usually communicates with the server over a language-agnostic standard network protocol like HTTP/S, usually receives data in a standard format like JSON, and, in contrast to direct database access, typically doesn't know what data store backs the API. 
+
+Of the above styles, [more often than not](https://rapidapi.com/learn/trends/2022-state-of-apis-what-developers-are-saying) API developers settle on HTTP-based REST APIs for one or more of the following reasons:
+
+1. Popularity: more developers know how to use REST APIs and are using them in production compared to other API technologies. E.g., Rapid API's [2022 State of APIs](https://rapidapi.com/learn/trends/2022-state-of-apis-what-developers-are-saying) reports 69.3% of survey respondents using REST APIs in production, well above the percentage using alternatives like gRPC (8.2%), GraphQL (18.6%), or webhooks (34.6%).
+2. Scale: REST's caching and statelessness properties provide a clear path to scale and client performance optimization. REST naturally lends itself to supporting many clients issuing many concurrent requests.
+3. Maturity: 20+ years of REST API adoption have resulted in a rich tooling ecosystem
+   * For API development & deployment. E.g., [OpenAPI](https://swagger.io/specification/) and its [tool](https://github.com/OpenAPITools/openapi-generator) for generating client bindings, API management solutions like [Kong Konnect](https://konghq.com/products/kong-konnect) to handle security and rate limiting.
+   * For API consumption. E.g., [Postman](https://www.postman.com/) for learning and testing APIs, and mature client libraries like [Python Requests](https://requests.readthedocs.io/en/latest/) for building applications that leverage REST APIs.
+4. Verb flexibility: while we are talking about _reading_ up-to-date data in this demo, many applications also need to create, update, and delete data too! HTTP-based REST services can use the same tools to develop and deploy all of the verbs. Some of the API patterns above like streaming are really only geared toward reading. While some mature engineering organizations offer APIs that span API styles (e.g., X exposes [streaming](https://developer.twitter.com/en/docs/twitter-api/enterprise/compliance-firehose-api/overview) and [REST](https://developer.twitter.com/en/docs/twitter-api/users/lookup/introduction) APIs), doing so comes at higher cost.
+
+Regardless of why data is exposed via REST API, the bottom line is:
+
+### As application developers, we don't always get to choose how to consume data. We have to use what API providers offer, and, more often than not, they give us REST APIs.
```

Comment:
I got excited here thinking that I'd learn to create a REST connector... maybe a link out to a tutorial like that could be helpful

Reply:
connector... wouldn't expect to learn how to make a REST API from Confluent